alpaca_trade_api/rest.py

```diff
@@ -546,7 +546,7 @@ def get_last_quote(self, symbol: str) -> Quote:
 
     def _data_get(self,
                   endpoint: str,
-                  symbol: str,
+                  symbol: Union[str, List[str]],
```

Comment:
symbol is probably not the best name for a list.
symbols is not the best too but better than symbol.

Reply:
renamed to `symbol_or_symbols`